File: Data Structures & Algorithms/append-characters-to-string-to-make-subsequence/submission-3.py
class Solution:
    def appendCharacters(self, s: str, t: str) -> int:
        T, S = 0, 0
        while T < len(t) and S < len(s):
            if s[S] == t[T]:
                S += 1
                T += 1
            else:
                S += 1
        return len(t) - T

File: Data Structures & Algorithms/is-anagram/submission-3.js
class Solution {
    /**
     * @param {string} s
     * @param {string} t
     * @return {boolean}
     */
    isAnagram(s, t) {
        let sSorted = s.split('').sort().join('')
        let tSorted = t.split('').sort().join('')

        if (sSorted === tSorted) return true
        return false
    }
}
